.kitchen.yml

```diff
@@ -1,6 +1,8 @@
 ---
 driver:
   name: vagrant
+  vagrantfiles: 
+      - vagrantfile.rb
```

Comment:
Hi Nathan
I don't think I can point test kitchen at an entire vagrantfile, as it's just added into the ephemeral test vagrantfile as a require and therefore needs to have the .rb extension. 
Do we still need the original vagrantfile, or should we just use test kitchen?

What do you think?

Sent from my iPhone

> On 20 Apr 2015, at 15:18, Nathan Lee <notifications@github.com> wrote:
> 
> In .kitchen.yml:
> 
> > @@ -1,6 +1,8 @@
> >  ---
> >  driver:
> >    name: vagrant
> > +  vagrantfiles: 
> > +      - vagrantfile.rb
> Just curious. Why the additional vagrantfile? You can add ruby to the regular Vagrantfile. Feel free to modify that one if you need to.
> 
> —
> Reply to this email directly or view it on GitHub.
>